scripts/sysman/common.yml

```diff
@@ -214,6 +214,9 @@ etors:
     - name: POWER_DOMAIN_EXP_PROPERTIES
```

Comment:
with the new changes, POWER_DOMAIN_EXP_PROPERTIES doesnt have a version anymore. Please re-add 1.8 for it.

Reply:
ah issue with merge conflict resolution. fixing

Reply:
fixed